pkg/apis/deviceplugin/v1/dlbdeviceplugin_webhook.go

```diff
@@ -33,7 +33,7 @@ var (
 	// dlbdevicepluginlog is for logging in this package.
 	dlbdevicepluginlog = logf.Log.WithName("dlbdeviceplugin-resource")
 
-	dlbMinVersion = version.MustParseSemantic(imageMinVersion)
+	dlbMinVersion = version.MustParseSemantic(controllers.ImageMinVersion)
```

Comment:
would it make sense to call version.MustParseSemantic once in controllers module?